### DISCUSSION TITLE
板子問題

### DISCUSSION THREAD

**Discussion by nthuyouwei** [bol-edu/HLS-SOC-Discussions#218] [Q&A]
我今天加入fir上板子 ，不小心在program時讓系統hold住，按下restart後kernel就連不上線了
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/145022311/785d7bd9-7299-4bba-937b-f24e0b8c7a00)
同時歸還後要再借會發現變成unknown，不知道有沒有辦法解決這個問題
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/145022311/dec34a99-30f1-4b60-9261-5684f13c9f24)

**Comment by Lucashien** [comment]
我們前幾天在測試時也遇到這問題，我們是多試了幾個板子才不會一直變unknown
另外，我們在測試時把最後一個run py給註解掉後進行測試，比較可以避免restart後連不上kernal的情況

給你參考一下我們的狀況嘿

**Comment by nthuyouwei** [comment]
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/145022311/b5f718b9-c562-4574-8d6e-7cfb3af9cdcb)
因為如果找不到解決辦法，再接者下去可能就沒板子用了.....

**Reply by john-1109** [reply]
同學你好，
如果pynq 出現 unknown，必須重啟 server 才能解決，但同學沒辦法自行重啟，所以鼓勵同學盡量使用 kv260 的板子。可以從 tcl 檔案裡面更改，或是太多 unknown 的話，我這邊可以手動幫同學重啟

**Reply by john-1109** [reply]
補充一下，每日5-6點 server 也會自動重啟一次，這時候 available 的板子會比較多
